lib/rubocop/cop/factory_bot/association_style.rb

```diff
@@ -0,0 +1,206 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module FactoryBot
+      # Use a consistent style to define associations.
+      #
+      # @safety
+      #   This cop may cause false-positives in `EnforcedStyle: explicit`
+      #   case. It recognizes any method call that has no arguments as an
+      #   implicit association but it might be a user-defined trait call.
+      #
+      # @example EnforcedStyle: implicit (default)
+      #   # bad
+      #   association :user
+      #
+      #   # good
+      #   user
+      #
+      # @example EnforcedStyle: explicit
+      #   # bad
+      #   user
+      #
+      #   # good
+      #   association :user
+      #
+      #   # good (NonImplicitAssociationMethodNames: ['email'])
```

Comment:
I'd love to see some do﻿﻿﻿cs when it will be useful. It is to ignore sequences to my best memory, right?

Reply:
Yes, this is an option that is intended to be used when a global sequence is defined and used as an implicit attribute. It is indeed hard to imagine how to use it, so I will add some more esample code 👍 

https://github.com/thoughtbot/factory_bot/blob/4a37cb64090d6354a719e8c5ef73653f5d242017/GETTING_STARTED.md#sequences